File: 8.7.2025/factorial.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        System.out.println("Enter a number:");
        int n = sc.nextInt();
        int fact = 1;
        for(int i=1;i<=n;i++){
            fact = fact *i;
        }
        System.out.printf("Factorial of %d is %d",n,fact);
    }
}

File: 8.7.2025/Factorial.c
#include <stdio.h>
int main() {
    int n;
    printf("Enter a number:\n");
    scanf("%d",&n);
    int fact = 1;
    for(int i=1;i<=n;i++){
        fact = fact *i;
    }
    printf("Factorial of %d is %d", n,fact);
}
